addons/analytic/models/analytic_plan.py

```diff
@@ -21,7 +21,8 @@ def _default_color(self):
         'account.analytic.plan',
         string="Parent",
         ondelete='cascade',
-        domain="[('id', '!=', id), ('company_id', 'in', [False, company_id])]",
+        domain="[('id', '!=', id), ('company_id', 'in', [False, company_id]), "
+        "'!', ('id', 'child_of', id)]",
```

Comment:
Good catch :)
However I don't understand why you wrote this additional condition like that ?
Either do it in one line
```suggestion
        domain="[('id', '!=', id), ('company_id', 'in', [False, company_id]), '!', ('id', 'child_of', id)]",
```
or in multiple lines (better imo, it's easier to read):
```suggestion
        domain="[
            ('id', '!=', id),
            ('company_id', 'in', [False, company_id]),
            '!', ('id', 'child_of', id),
        ]",
```

Reply:
Noted, thanks!